extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -88,8 +88,10 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     try {
       const { raw } = await this.view.emailProvider.msgGet(msgId, 'raw', progress => this.setQuoteLoaderProgress(progress));
       this.setQuoteLoaderProgress('processing...');
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      const decoded = await Mime.decode(Buf.fromBase64UrlStr(raw!));
+      if (typeof raw === 'undefined') {
+        return undefined;
+      }
```

Comment:
Maybe we can simplify like below? This will check if raw is empty/null/undefined too though.
```ts
if (!raw) {
  return;
}
```

Reply:
Thanks, Ioan, for the remarks. I will apply the suggested code simplification.

Reply:
Your welcome. You actually don't need `return undefined` too. `return` would be sufficient